libraries/common/subscriptions/helpers/handleLinks.js

```diff
@@ -143,11 +137,35 @@ export const isShopLink = (location) => {
   return true;
 };
 
+/**
+ * Check if the given pathname is a protector route.
+ * @param {string} location The location to check.
+ * @return {boolean}
+ */
+export const isProtector = location => Array.from(authRoutes.getAll().values()).includes(location);
```

Comment:
Please put this inside the AuthRoutes class and name it `isProteced()`.
So you can say: `authRoutes.isProteced(location)`. This will be a much cleaner solution.

Reply:
i've put it there because there was already a function called `getProtector` within this module.
but you're right. i've now moved both into the `AuthRoutes` class since i also think that those helpers belong there.